### PR TITLE
Task-List-Container component close #1484

### LINKED ISSUE #1484: Task lists do not have any minimum height
# Problem

The task lists need a minimum height of at least `10px` or something, otherwise when they are empty, a task card cannot be dragged into it.

We should also probably add a component test to ensure the drop zone portion of the task has some minimum height

### PR DESCRIPTION
**Pull request by snewcomer**
- [x] add task-list height of 100%
- [x] test

Fixes #1484

### CODE REVIEW

File: tests/acceptance/task-list-test.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ test('member can assign/reassign/unassign tasks to user', function(assert) {
   });
 
   andThen(() => {
+    assert.ok(find('.task-list').css('height') > '500px', 'the task-list has height of 100%');
```
Comment (snewcomer) [inline tests/acceptance/task-list-test.js]: @JoshSmith not sure I am a huge fan of this...Any other thoughts on testing the height if you still want it tested?

File: tests/acceptance/task-list-test.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ function createUserWithRole(project, role) {
   return user;
 }
 
-test('member can assign/reassign/unassign tasks to user', function(assert) {
-  assert.expect(7);
+test('scott member can assign/reassign/unassign tasks to user', function(assert) {
```
Comment (snewcomer) [inline tests/acceptance/task-list-test.js]: still need to finish this.

File: app/styles/components/task-list-cards.scss [hunk 3]
```diff
@@ -30,6 +30,6 @@
   }
 }
 
-.task-list-cards {
-  height: 100%;
-}
+// .task-list-cards {
+//   height: 100%;
+// }
```
Comment (snewcomer) [inline app/styles/components/task-list-cards.scss]: Don't this this was needed.

**Comment by snewcomer** [comment]
Closing in favor of spiking out https://github.com/timrourke/ember-cli-shopify-draggable-shim
